fineract-provider/src/main/java/org/apache/fineract/cob/api/InternalLoanAccountLockApiResource.java

```diff
@@ -76,4 +77,23 @@ public Response placeLockOnLoanAccount(@Context final UriInfo uriInfo, @PathPara
         loanAccountLockRepository.save(loanAccountLock);
         return Response.status(Response.Status.ACCEPTED).build();
     }
+
+    @POST
```

Comment:
Please update with optionally provide error the existing `placeLockOnLoanAccount` instead of creating a new one.

Reply:
Fixed